### ISSUE TITLE
Allow webviews to use `vscode-high-contrast-light`

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#148057]
As discussed in #145179, we should let webviews make decisions based on the high contrast theme type.

Proposed by @mjbvz:

> With my proposal, the classes in the webview would be:
>
> high contrast dark: `vscode-high-contrast`
> high contrast light: `vscode-high-contrast vscode-high-contrast-light`

cc @hawkticehurst

**Comment by alexr00** [comment]
How should this be verified?

**Comment by rchiodo** [comment]
I _think_ this can be verified by checking the style of the body in a webview?

Here's one in stable with high-contrast light theme enabled:

![image](https://user-images.githubusercontent.com/19672699/176551515-d32a0e68-904e-49b5-ab77-bc7f4e4b9b97.png)

Here's the same one in insiders with same theme enabled:

![image](https://user-images.githubusercontent.com/19672699/176551572-23ac3def-24c5-46d2-8982-2e947c0a3e3a.png)

It would allow CSS rules based on the body having that class.

**Comment by rchiodo** [comment]
So something like so?

```
body.vscode-high-contrast-list .button{
    background-color: 'red'
}
```

**Comment by rchiodo** [comment]
/verified

Yeah that works for me. I used the webview-view-sample.

**Comment by hawkticehurst** [comment]
Yeah, in the webview ui toolkit [we look for the value set on the `data-vscode-theme-kind` attribute](https://github.com/microsoft/vscode-webview-ui-toolkit/blob/main/src/utilities/theme/applyTheme.ts#L38) so, hopefully, the vscode-high-contrast-light theme would be reflected there as well.

**Comment by hawkticehurst** [comment]
Which actually now that I look at @rchiodo's screenshots more closely, it already is being reflected on the `data-vscode-theme-kind` attribute

**Comment by daviddossett** [comment]
Also confirmed this works when working on a fix for the webview toolkit last week 👍
